druid/examples/sub_window.rs

```diff
@@ -145,11 +145,9 @@ impl<T, W: Widget<T>> Controller<T, W> for TooltipController {
                             WindowConfig::default()
                                 .show_titlebar(false)
                                 .window_size_policy(WindowSizePolicy::Content)
-                                .set_level(WindowLevel::Tooltip)
+                                .set_level(WindowLevel::Tooltip(ctx.window().clone()))
                                 .set_position(
-                                    ctx.window().get_position()
-                                        + window_pos.to_vec2()
-                                        + cursor_size.to_vec2(),
+                                    (window_pos.to_vec2() + cursor_size.to_vec2()).to_point(),
```

Comment:
Apologies for being vague. Location of tooltip is way off target.
![image](https://user-images.githubusercontent.com/6751090/134032835-63f94717-6673-415c-a3cd-509f9d3fb8f1.png)

Reply:
Could it be some DPI scaling? I cant really test but ill check if I can spot something related to that.

Reply:
The problem at least on Windows is that the cursor position received in `TooltipController` is in **window coords**. So we need to go from window coords to screen coords because screen coords is what Windows expect when you create a tooltip.

Reply:
Yes that conversion happens in win/window.rs. So that should be handled. Instead of adding the parent window size in the users code we do it in druid_shell.

Reply:
Not sure if it's handled. Otherwise this should work??
Solution to this problem is simple
```
let tooltip_position_in_window_coordinates = (window_pos.to_vec2() + cursor_size.to_vec2()).to_point();
let final_position = self.window().get_position() + tooltip_position_in_window_coordinates .to_vec2();
```
Then pass `final_position` to sub window. This works for 100% scaling but not for other scaling factors. I was about to open a PR to fix all this. 

On a different topic, what should we do when we fix/enhance one platform but cant port changes to other platforms or test it? Ditto porting might not be even the right thing. I'll open topic in Zulip.

Reply:
No because those coordinates are in global space while they should be in window space. they are converted to global space by `druid_shell` not the client code. This was part of the change, so it should work. Maybe it cant get the parents state? that could be an issue.

Reply:
The solution i gave in previous comment is basically the old code. That was dumb on my part. lol. The previous code didnt make much sense to me.

Reply:
Where in shell is this conversion happening?

Reply:
https://github.com/linebender/druid/pull/1919/files#diff-1a1a9e4151b4aa7f5d4959110f78d6f99d7fa46ac42ce3d05ac2992ab3dfe6e9R1852

Reply:
Check https://xi.zulipchat.com/#narrow/stream/147926-druid/topic/Tooltip.20broken.20in.20Windows/near/254600089